copy_props.ps1

```diff
@@ -0,0 +1,25 @@
+###
+### Sync Directory.Build.props in project folder with "main" config/Directory.Build.props file
+### Project should reference to any Steeltoe package with Version=$(SteeltoeVersion).
+### 
+### TODO: add shell script for Linux
```

Comment:
Powershell is available for Linux so this TODO is effectively completed.

You can make it an executable script on Linux by:
* converting file line endings from DOS to UNIX
* set the file's execute bit (may not be able to do this from Windows?)
* making the first line of the script:
```
#/usr/bin/env pwsh
```

If you don't have access to a Linux, or are uncomfortable making these mods, I can make them after the PR is accepted.